docs/guide/cache/content_aware_cache.md

```diff
@@ -293,3 +350,253 @@ bin/console fos:httpcache:invalidate:tag ez-all
+```
+
+Some notes about each of these parameters:
+- `-IXGET`, one of many ways to tell curl that we want to send a GET request, but we are only interested in outputting the headers
+- `--resolve www.staging.foobar.com.us-2.platformsh.site:443:1.2.3.4`
+  - We tell curl not to do a DNS lookup for `www.staging.foobar.com.us-2.platformsh.site`. We do that because in our case
+    that will resolve to the Fastly endpoint, not our origin (nginx)
+  - We specify `443` because we are using `https`
+  - We provide the IP of the nginx endpoint at platform.sh (`1.2.3.4` in this example)
+- `--header "Surrogate-Capability: abc=ESI/1.0"`, strictly speaking not needed when fetching the user-context-hash, but this tells [[= product_name =]] that client understands ESI tags.
+  It is good practice to always include this header when imitating the HTTP Cache.
+- `--header "accept: application/vnd.fos.user-context-hash"` tells [[= product_name =]] that the client wants to receive the user-context-hash
+- `--header "x-fos-original-url: /"` is required by the fos-http-cache bundle in order to deliver the user-context-hash
+- `https://your-page-blah-blah.us-2.platformsh.site/_fos_user_context_hash` : here we use the hostname we earlier told
+  curl how to resolve using `---resolve`. `/_fos_user_context_hash` is the route to the controller that are able to
+  deliver the user-context-hash.
+- You may also provide the session cookie (`--cookie ".....=....") for a logged-in-user if you are interested in
+  the x-user-context-hash for a different user but anonymous
+
+The output for the given command should look similar to this:
+```
+    HTTP/1.1 200 OK
+    Server: nginx/1.20.0
+    Content-Type: application/vnd.fos.user-context-hash
+    Transfer-Encoding: chunked
+    Connection: keep-alive
+    X-User-Hash: daea248406c0043e62997b37292bf93a8c91434e8661484983408897acd93814
+    Cache-Control: max-age=600, public
+    Date: Tue, 31 Aug 2021 13:35:00 GMT
+    Vary: Origin
+    Vary: cookie
+    Vary: authorization
+    X-Cache-Debug: 1
+    Surrogate-Key: ez-user-context-hash ez-all fos_http_cache_hashlookup-
+```
+
+The header `X-User-Hash` is the one we are interested in here, but you may also note the `Surrogate-Key` which
+holds the [cache tags](#understanding-cache-tags).
+
+### Fetching the HTML response
+
+Now that we have the user-context-hash, we can ask origin for the actual resource we are after:
+
+```bash
+    $ curl -IXGET --resolve www.staging.foobar.com.us-2.platformsh.site:443:1.2.3.4 --header "Surrogate-Capability: abc=ESI/1.0" --header "x-user-hash: daea248406c0043e62997b37292bf93a8c91434e8661484983408897acd93814" https://www.staging.foobar.com.us-2.platformsh.site/
+```
+
+The output :
```

Comment:
```suggestion
Nowyou have the user-context-hash, and you can ask origin for the actual resource you are after:

```bash
    $ curl -IXGET --resolve www.staging.foobar.com.us-2.platformsh.site:443:1.2.3.4 --header "Surrogate-Capability: abc=ESI/1.0" --header "x-user-hash: daea248406c0043e62997b37292bf93a8c91434e8661484983408897acd93814" https://www.staging.foobar.com.us-2.platformsh.site/
```

The output :

```

Reply:
@vidarl by word: actual you mean the current one or the real one?

Reply:
I don't understand your question.

But I guess "Nowyou" ---> "Now you" ?

Reply:
@vidarl In the question I mean: does the word 'actual' stands for the "existing in fact or reality" or for "current, up to date"